sdk/keyvault/keyvault-secrets/README.md

```diff
@@ -85,16 +86,35 @@ Use the [Azure Cloud Shell](https://shell.azure.com/bash) snippet below to creat
   az keyvault show --name <your-key-vault-name>
   ```
 
+## Key concepts
+
+- The **SecretsClient** is the primary interface to interact with Azure Key Vault
```

Comment:
The **Secrets client** is the...

Reply:
gotcha! I thought you were referring to the actual name of the class, but I understand now